cmd/ostracon/commands/kms_test.go

```diff
@@ -0,0 +1,64 @@
+package commands
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/line/ostracon/crypto"
+	"github.com/line/ostracon/crypto/ed25519"
+	"github.com/line/ostracon/libs/log"
+	"github.com/line/ostracon/privval"
+)
+
+func WithMockKMS(t *testing.T, dir, chainID string, f func(string, crypto.PrivKey)) {
```

Comment:
Should move into `privval` or `node` package.

Reply:
This file has been moved to `privval/test_util.go` because Golang’s test builds couldn't reference common functions defined in `*_test.go` outside the package.

Note that `utils_test.go` also exists in the same directory, Here is a test case for `utils.go`.